lib/pbench/server/__init__.py

```diff
@@ -45,6 +45,10 @@ class PbenchServerConfig(PbenchConfig):
     for the pbench server environment.
     """
 
+    # Define a fallback default for dataset maximum retention, which we expect
+    # to be defined in pbench-server-default.cfg.
+    MAXIMUM_RETENTION_DAYS = 3650
```

Comment:
If we expect it to come from pbench-server-default why do we define it here?

Reply:
This is a pattern followed in many other places. We've discussed before the idea of doing config file validation on startup; there's probably even an issue on that, although I won't bother to look it up right now.